frontend/src/components/core/Sidebar/SidebarNav.tsx

```diff
@@ -15,22 +15,27 @@
  * limitations under the License.
  */
 
-import React, { ReactElement, useState } from "react"
+import React, { ReactElement, useState, useRef, useCallback } from "react"
+import Icon from "src/components/shared/Icon"
+import { useIsOverflowing } from "src/lib/Hooks"
+import { ExpandMore, ExpandLess } from "@emotion-icons/material-outlined"
```

Comment:
Ah, yeah I also generally try to group imports into third party, internal (absolute), and internal (relative) imports, and within each group order imports alphabetically. I must have missed reordering this when I was cherry-picking commits onto this branch

I don't think we have any official style requirement around import ordering at the moment, though. We could probably configure our linter to enforce this, but since imports are a bit inconsistent in the codebase, it'd be a chore to fix all of the parts where we're inconsistent.

I'll fix the ordering in this specific case